.github/workflows/e2e.yml

```diff
@@ -33,7 +33,7 @@ jobs:
       packages: write
     steps:
       - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
-      - if: ${{ !inputs.self-hosted }}
+      - if: (!inputs.self-hosted)
```

Comment:
Do we need the parentheses here?

Reply:
yes, otherwise it's interpreted as yaml foo